src/page/GroupCreate.tsx

```diff
@@ -0,0 +1,403 @@
+import { useRef, useState } from 'react';
+import styled from 'styled-components';
+import {
+  IcAddPhoto,
+  IcArrowBottomGray,
+  IcSecretGray,
+  IcUnlockGray,
+} from '../assets';
+import CommonButton from '../common/CommonButton';
+import PageLayout from '../components/PageLayout/PageLayout';
+import { PLACEHOLDER } from '../constants/CommonTextarea/textareaConst';
+
+const GroupCreate = () => {
+  const [isPublicGroup, setIspublicGroup] = useState(true);
+  const [previewImage, setPreviewImage] = useState<string | null>(null);
+  const [isActive, setIsActive] = useState(false);
+  const memberCountRef = useRef<HTMLInputElement>(null);
+
+  const handlePublicClick = () => {
+    if (!isPublicGroup) setIspublicGroup(true);
+  };
+
+  const handleSecretClick = () => {
+    if (isPublicGroup) setIspublicGroup(false);
+  };
```

Comment:
P5:  아래처럼 하나의 함수로 통합할 수 있어요 !
```suggestion
  const handleClickButton = () => {
    setIspublicGroup(!isPublicGroup);
  };
```

Reply:
아아 넵 하다가 까먹고 수정을 못했네요 바로 수정하겠습니다!